pkg/apis/core/types.go

```diff
@@ -5581,6 +5581,10 @@ type NodeStatus struct {
 	// +featureGate=SupplementalGroupsPolicy
 	// +optional
 	Features *NodeFeatures
+	// DeclaredFeatures represents the declared features of a node.
+	// +featureGate=NodeDeclaredFeatures
+	// +optional
```

Comment:
I think we put listType annotations on the internal types? I'm not sure what that does, but it's consistent.

Reply:
There's some inconsistency on this, but since it doesn't do anything on the internal types, it might be preferable to omit it. :+1: